app/models/project.rb

```diff
@@ -0,0 +1,4 @@
+class Project < ApplicationRecord
+  has_many :contributions
```

Comment:
You can add the `dependent: :destroy` for this kind of relations

Reply:
@Sanchezdav we want to maintain the data once a member left the organization, so it is going to be stored with historical purposes

Reply:
if you use something like `paranoid` gem to delete just on logic level and if you try to delete a project for example the contributions should be remove it for logic as well with the `dependent: :destroy`